subsys/emds/Kconfig

```diff
@@ -19,6 +20,40 @@ config EMDS_SECTOR_COUNT
 	help
 	  Number of sectors used for the emergency data storage area
 
+config EMDS_THREAD_STACK_SIZE
+	int "Stack size for the emergency data storage thread"
+	default 500
+	help
+	  Size of the stack that is initialized by the emergency data storage
+
+config EMDS_THREAD_PRIORITY
+	int "Priority for the emergency data storage thread"
+	default 0
+	help
+	  Cooperative priority for the emergency data storage thread. This will
+	  be used through K_PRIO_COOP(x), that means higher value gives lower
+	  priority.
+
+config EMDS_FLASH_TIME_WRITE_ONE_WORD_US
+	int "Time to write one word into flash"
+	default 41
+	help
+	  Max time to write one word (4 bytes) in flash (in microseconds)
+
+config EMDS_FLASH_TIME_ENTRY_OVERHEAD_US
+	int "Time to schedule write of one entry"
+	default 1410 if BT
+	default 42
+	help
+	   Max time to prepare the write of each entry (in microseconds)
+
+config EMDS_FLASH_TIME_BASE_OVERHEAD_US
+	int "Time to schedule the store process"
+	default 5000 if PARTIAL_ERASE
```

Comment:
You got some compliance error on the `PARTIAL_ERASE` clause here that must be resolved